manager/state/raft/raft.go

```diff
@@ -461,14 +457,16 @@ func (n *Node) Run(ctx context.Context) error {
 			// If we are the only registered member after
 			// restoring from the state, campaign to be the
 			// leader.
-			if !n.restored {
-				// Node ID should be in the progress list to Campaign
-				if len(n.cluster.Members()) <= 1 {
+			if n.campaignWhenAble {
+				members := n.cluster.Members()
```

Comment:
Actually, I see. Can you pls change comment, so it'll reflect that we start campaign also on inital cluster start?